server/Arcadia.Assistant.Calendar/SickLeave/SendEmailSickLeaveActor.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Arcadia.Assistant.Calendar.SickLeave.Events;
+using System;
+using System.Configuration;
+using System.Threading;
+using Akka.Actor;
+using MimeKit;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Gmail.v1;
+using Google.Apis.Services;
+using Google.Apis.Gmail.v1.Data;
+
+namespace Arcadia.Assistant.Calendar.SickLeave
+{
+    public class SendEmailSickLeaveActor : UntypedActor
+    {
+        protected override void OnReceive(object message)
+        {
+            switch (message)
+            {
+                case SickLeaveIsApproved ev:
+                    SendEmail(ev);
+                    break;
+            }
+        }
+
+        private static void SendEmail(SickLeaveIsApproved Event)
+        {
+            var clientId = ConfigurationManager.AppSettings["ClientId"];
+            var clientSecret = ConfigurationManager.AppSettings["ClientSecret"];
+            try
+            {
+                var credential = GoogleWebAuthorizationBroker.AuthorizeAsync(
+                    new ClientSecrets
+                    {
+                        ClientId = clientId,
+                        ClientSecret = clientSecret,
+                    },
+                    new[] { GmailService.Scope.GmailCompose },
+                    "user",
+                    CancellationToken.None).Result;
+
+                var service = new GmailService(new BaseClientService.Initializer
```

Comment:
you should not use GmailService, we will use our own SMTP server in production, not Gmail. Gmail supports SMTP, that's what needs to be used (its settings - https://www.lifewire.com/what-are-the-gmail-smtp-settings-1170854 )